src/page/gjennomforing-detaljer/endringsmeldinger/endringsmelding/Endringsmelding.tsx

```diff
@@ -52,22 +59,26 @@ export const Endringsmelding = ({ endringsmelding, onFerdig, className }: Endrin
 						>
 							Ferdig
 						</Button>
-					)
-					: (<BodyShort className={classNames(styles.moveRight, styles.gray)}>Ferdig</BodyShort>)
-				}
-			</PanelLinje>
+					</PanelLinje>
+				)
+				: (
+					<PanelLinje>
+						<BodyShort className={classNames(styles.moveRight, styles.gray)}>Ferdig</BodyShort>
```

Comment:
Litt personlig preferanse, men med SCSS moduler pr komponent så foretrekker jeg mest 1 klasse pr komponent for å style den, med unntak av klasser fra designsystemet. Tailwind-style klasser fungerer også bra, men de hører kanskje mer hjemme i en (eller flere) felles style-fil(er) som kan gjenbrukes i applikasjonen.